.buildkite/scripts/pipelines/pull_request/pipeline.js

```diff
@@ -104,8 +104,8 @@ const uploadPipeline = (pipelineContent) => {
       pipeline.push(getPipeline('.buildkite/pipelines/pull_request/osquery_cypress.yml'));
     }
 
-    if (await doAnyChangesMatch([/^x-pack\/plugins\/synthetics/])) {
-      pipeline.push(getPipeline('.buildkite/pipelines/pull_request/synthetics_plugin.yml'));
+    if (await doAnyChangesMatch([/^x-pack\/plugins\/observability/])) {
+      pipeline.push(getPipeline('.buildkite/pipelines/pull_request/observability_plugin.yml'));
```

Comment:
Did you mean to replace this or should it be in addition to the pipeline for `/^x-pack\/plugins\/synthetics/`? If the replacement was intentional I think you need to remove the old pipeline at https://github.com/elastic/kibana/blob/main/.buildkite/pipelines/pull_request/synthetics_plugin.yml and it's scripts, etc.

Reply:
oops you are right, this should have been addition